environments/production/app-processes.yml

```diff
@@ -3,7 +3,7 @@ django_command_prefix: ''
 celery_command_prefix: ''
 gunicorn_workers_static_factor: 0
 gunicorn_workers_factor: 2
-formplayer_memory: "30g"
+formplayer_memory: "15g"
```

Comment:
Just to clarify, this setting applies to both `formplayer_a` and `formplayer_b`, but you're going to just target the runs to the b's?

Reply:
That's right, yeah I should have said that. That said, assuming there are no issues with formplayer_b's setup, it would probably be safe to roll this out to other machines as well